PiwikTracker.php

```diff
@@ -1163,14 +1163,16 @@ public function setIp($ip)
      *
      * A User ID can be a username, UUID or an email address, or any number or string that uniquely identifies a user or client.
      *
-     * @param string $userId Any user ID string (eg. email address, ID, username). Must be non empty. Set to false to de-assign a user id previously set.
+     * Call resetUserId() to de-assign a user id previously set.
+     *
+     * @param string $userId Any user ID string (eg. email address, ID, username). Must be non empty.
```

Comment:
FYI: The doc may need to be adjusted as when passing `false` it does something possibly unexpected even though it says "Must not be empty".